toolbox/repo/validate_role_vars_used.py

```diff
@@ -0,0 +1,78 @@
+#! /usr/bin/env python
+
```

Comment:
Right now the only documentation for what this script does is in the changelog. I think it would be helpful to add a comment to the top of the file that explains in short what it does.

Same goes for the other script

Reply:
ack, I'll even add a `-h` and `--help`!